items.md

```diff
@@ -137,6 +137,172 @@ Example: <https://dspace7.4science.it/dspace-spring-rest/#https://dspace7.4scien
 
 It returns the collection where the item belong to
 
+### Mapping Collections
+**GET /api/core/items/<:uuid>/mappingCollections**
+
+Example:
+```json
+    "mappingCollections":
+    [
+      {
+        "id": "16a4b65b-3b3f-4ef5-8058-ef6f5a653ef9",
+        "uuid": "16a4b65b-3b3f-4ef5-8058-ef6f5a653ef9",
+        "name": "Bachelor theses",
+        "handle": "123456789/5287",
+        "metadata": [
+          {
+            "key": "dc.description.abstract",
+            "value": "",
+            "language": null
+          },
+          {
+            "key": "dc.title",
+            "value": "Bachelor theses",
+            "language": null
+          }
+        ],
+        "type": "collection",
+        "_links": {
+          "license": {
+            "href": "https://dspace7-internal.atmire.com/rest/api/core/collections/16a4b65b-3b3f-4ef5-8058-ef6f5a653ef9/license"
+          },
+          "exportToZip": {
```

Comment:
This is something new not yet discussed (BTW I would recommend to use content-negotiation for the export)

Reply:
This was not supposed to be included here. It has been removed now